### PR TITLE
[CLOUDSTACK-10039] Adding allocated IOPS to storage pool response

### PR DESCRIPTION
**Pull request by syed**
This change adds `allocatediops` to the `ListStoragePool` API. This applies to managed storage where we have a guaranteed minimum IOPS set. This is useful for monitoring if we have reached the IOPS limit on a storage cluster.

cc: @mike-tutkowski

### CODE REVIEW

**Comment by mike-tutkowski** [comment]
This LGTM, @syed - thanks!

**Comment by nitin-maharana** [comment]
Code LGTM.

**Comment by mike-tutkowski** [comment]
I guess one comment I have is that technically we are supposed to have a JIRA ticket for each PR. Perhaps you could open an enhancement ticket and reference it here, @syed - thanks!

File: api/src/org/apache/cloudstack/api/response/StoragePoolResponse.java [hunk 1]
```diff
@@ -288,6 +292,10 @@ public void setCapacityIops(Long capacityIops) {
         this.capacityIops = capacityIops;
     }
 
+    public void setAllocatedIops(Long allocatedIops) {
```
[the remark this answers is not in the document]
Comment (syed) [reply api/src/org/apache/cloudstack/api/response/StoragePoolResponse.java]: Hi @nitin-maharana. Thanks for the review. The getter is not needed as there is no one reading the vlaue.

**Comment by syed** [comment]
@mike-tutkowski I had opened the ticket in jira a while back. I've edited the PR to add the JIRA ticket

**Comment by mike-tutkowski** [comment]
Thanks, @syed!

**Comment by borisstoyanov** [comment]
@blueorangutan package

**Comment by borisstoyanov** [comment]
@syed could this be a related failure? 
```
Results :

Failed tests: 
  ConstantTimeBackoffTest.waitBeforeRetryWithInterrupt:48 null

Tests run: 266, Failures: 1, Errors: 0, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Apache CloudStack Developer Tools - Checkstyle Configuration  SUCCESS [6.800s]
[INFO] Apache CloudStack ................................. SUCCESS [13.782s]
[INFO] Apache CloudStack Maven Conventions Parent ........ SUCCESS [5.497s]
[INFO] Apache CloudStack Framework - Managed Context ..... SUCCESS [8.243s]
[INFO] Apache CloudStack Framework ....................... SUCCESS [0.080s]
[INFO] Apache CloudStack Framework - Certificate Authority  SUCCESS [1.388s]
[INFO] Apache CloudStack Utils ........................... FAILURE [49.796s]
```
will re-kick build @blueorangutan package

**Comment by borisstoyanov** [comment]
@blueorangutan test

**Comment by mike-tutkowski** [comment]
@borisroman Are we good to go with this PR? Should I merge it? Thanks!

**Comment by mike-tutkowski** [comment]
Are we good to go with this PR? Should I merge it? Thanks!

**Comment by rhtyd** [comment]
LGTM, merging this now.

**Comment by syed** [comment]
Thanks @rhtyd

**Pull request merged**